test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -2,12 +2,70 @@
 // Licensed under the MIT License.
 
 #include "pch.h"
+#include <KozaniManager_h.h>
+#include "winrt/Microsoft.Kozani.ManagerRuntime.h"
 
 namespace TB = ::Test::Bootstrap;
 namespace TP = ::Test::Packages;
 
+namespace KozaniManagerTestPackage
+{
+    constexpr PCWSTR c_PackageDirName = L"KozaniManager";
+    constexpr PCWSTR c_PackageFullName = L"KozaniManager_1.0.0.0_neutral__8wekyb3d8bbwe";
+}
+
 namespace Test::KozaniManagerRuntimeTests
 {
+    struct MyKozaniStatusCallback : winrt::implements<MyKozaniStatusCallback, IKozaniStausCallback, winrt::Windows::Foundation::IInspectable>
+    {
+#pragma region IKozaniStausCallback_methods
```

Comment:
Cute. Is it useful?

Reply:
Yes, this is how I validated the API E2E across processes actually works. It uncovered code that compiled but didn't work OOP due to missing proxy stub, etc.